### DISCUSSION TITLE
Proposal : Combine 2 game changes to keep balance (25.50 and Umbrella Fix)

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18289] [Feature requests]
I tried to increase the cap on rides and shop items from 20 dollars to 25.50, but the community wasn't interested. The benefit of this increase would be for certain coaster types that can in theory help out mega parks and attract guests. The community thought this was a cheap way to boost speed running, which I disagreed with since I believe speedrunning already utilizes lots of strats to get incredibly cheap victories. Also, speedrunning is not the most popular way to play openrct2, most people play it for management. My proposal is to bring back this 25.50 cap under the condition that the umbrellas are modified in a certain way: Have the maximum price that peeps will pay for umbrellas actually determined in game code, and set that maximum to between 14.50 and 25.50. In this way, users that go for 20 dollar umbrellas will be penalized by poorer guests, or less happy guests, but rewarded by guests that are very happy. This makes park management more important. In addition, setting umbrellas to 25.50 will mean that guests will run out of money and leave the park sooner unless there is a cash machine, and naturally less guests will buy umbrellas. In this way, we get the benefit of being able to charge more for giant giga coasters, or air powered vertical coasters, that are expensive to build in the first place, while umbrellas aren't simply used as an exploit in the game.

**Comment by Lastorder-DC** [comment]
I'm afraid but Nobody will read that long sentence.

So you're basically saying that RCT 20$ Limit should increased with prevent of "Umbrella abuse"(guests will buy umbrella if raining - even if its price is 20$).

**Comment by ZehMatt** [comment]
How is 20 not enough?

**Comment by 733737** [comment]
a range preserves the 20 dollar umbrella standard while removing it for the purpose of adding complexity to the ai that will provide more unique gameplay for inexperienced players and require new optimizations for professional players.

**Comment by Broxzier** [comment]
If players want to abuse a game mechanic to their advantage, that's up to them. I don't think the game should punish that. I think most casual players don't even know about it. As you say, expensive umbrellas has as downside that guests run out of cash sooner, and guests will only buy one umbrella anyway.
